ex2_hal/iris/hardware_interface/source/iris_bootloader_cmds.c

```diff
@@ -99,11 +70,11 @@ void iris_pre_sequence() {
  **/
 void iris_post_sequence() {
     /* End I2C transaction by doing end sequence*/
-    BOOT_LOW();
-    vTaskDelay(100);
-    POWER_OFF();
-    vTaskDelay(100);
-    POWER_ON();
+    IRIS_BOOT_LOW();
```

Comment:
It's not typical for a function call to be formatted the same as a macro replacement, and in fact this function is a bit hard to read with all the capitals that represent different things

Reply:
I will make these lower case, like other driver apis